jalbe0076/whats-cooking
# Feature/fetch recipes

### What is the feature?
- This PR will close #23 & close #24 & close #25
- Updated the project to fetch data from a network and removed the local data

### How you implemented the solution?
- Created a dynamic function to fetch API's 
- Used the `promiseAll` method to take all the promises and resolve them on page load
- Saved the resolved promises to global variables to replace the previous global variables

### Does it impact any other area of the project?
- It completely removes the local data, only the sample testing data was kept

### Any changes in the UI (Screenshots)?
- None

### How to test the feature (A test scenario or any new setup)?
-Pull the PR down to your local machine and run `npm start` in the terminal, play around on the site to ensure all previous implemented functionality of the code works as intended.
